Lesson_2.1/src/main/java/level_2/Cat.java

```diff
@@ -0,0 +1,26 @@
+package level_2;
+
+import java.util.Random;
+
+public class Cat implements Team {
+
+    private String name;
+
+    Random random = new Random();
```

Comment:
```suggestion
    private Random random = new Random();
```
также его можно использовать вместо math.random
аналогично для других классов